exercises.js

```diff
@@ -55,7 +56,7 @@ function setupToc(exercises) {
 function addDomElements(exercises) {
   for (let i = 0; i < exercises.length; i++) {
     let exercise = exercises[i];
-    let $nav = document.createElement("div");
+    let $nav = document.createElement("nav");
```

Comment:
there's a nav html element??? who knew!!? (not me anyway)

Reply:
BTW almost none of this html will remain in the React version as this DOM just doesn't make sense as is. But let's do what we can for now!